.github/workflows/test.yaml

```diff
@@ -43,6 +43,9 @@ jobs:
       - name: Install dependencies
         run: pnpm i
 
+      - name: Lint packages
+        run: pnpm lint:fix
```

Comment:
I assume you were expecting the `lint:fix` command to throw an error when there are lint problems. Did it work as expected when you ran the command locally, or did it silently fix things for you instead?

Reply:
Yes. It works for the examples on my computer.

See https://github.com/ckb-cell/rgbpp-sdk/blob/refactor/add-examples-eslint/examples/rgbpp/package.json#L9